webapp/components/edit_post_modal.jsx

```diff
@@ -91,16 +89,12 @@ export default class EditPostModal extends React.Component {
             return;
         }
 
-        Client.updatePost(
+        updatePost(
             updatedPost,
             () => {
                 loadPosts(updatedPost.channel_id);
```

Comment:
Could you move the `loadPosts` line into the success callback of the action itself?